page01_index.php

```diff
@@ -52,118 +53,114 @@
 		</div>
 
 
-
-		<br>
-		<br>
-		<br>
-		<br>
+		<div class="w3-black">
+			<div class="w3-container w3-content w3-padding-64" style="max-width:100%">
 
 
-		<div class="w3-panel w3-padding-large w3-margin-left w3-margin-right w3-blue">
+				<div class="w3-panel w3-padding-large w3-margin-left w3-margin-right w3-black">
 
 
-			<!-- slideshow for album covers  -->
-			<div class="w3-panel w3-padding-large w3-yellow" style="width:100%">
-				<div class="w3-container w3-large w3-pink">
-					<div class="w3-left-align"><p>Welcome To SoundWave <?php echo $User;?>!<br/></p></div>
+					<!-- slideshow for album covers  -->
+					<div class="w3-panel w3-padding-large w3-black" style="width:100%">
+						<div class="w3-container w3-large w3-black">
+							<div class="w3-left-align"><h2>Welcome To SoundWave <?php echo $User;?>!<br/></h2></div>
+							
+							<div class="w3-row-padding w3-padding-16 w3-center">
 
+								<div class="mySlides w3-third">
+									<img class="w3-round-large" src="images/placeholder.jpg" alt="Sandwich"   width="210" height="110" style="width:100%;max-width:230px">
+									<h4>Album name</h4>
```

Comment:
changed writing to h4, this makes the text white and good secondary titles for album covers